extensions/amp-form/0.1/test/test-form-dirtiness.js

```diff
@@ -98,6 +105,15 @@ describes.realWin('form-dirtiness', {}, env => {
       changeInput(textField, 'changed');
       expect(form).to.have.class(DIRTINESS_INDICATOR_CLASS);
     });
+
+    it('removes dirtiness class when its value matches the submitted value', () => {
```

Comment:
I wasn't sure to place the tests under `text field/textarea changes` or `#onSubmitSuccess`. Any preferences? @GoTcWang

Reply:
Do we need two tests for both text field and textarea, are they testing the same code?

If they are testing the same code, you can make them one and put it under `#onSubmitSuccess`, otherwise, under each input type seems to work better.